plugins/modules/nutanix_vm.py

```diff
@@ -466,108 +985,91 @@ def create_vm_spec(params, vm_spec):
                 }
             }
 
-    vm_spec["spec"]["cluster_reference"] = {"kind": "cluster", "uuid": params['cluster']}
+    return vm_spec, None
 
-    return vm_spec
 
+def update_vm_spec(params, current_vm_payload, client):
 
-def update_vm_spec(params, vm_data):
```

Comment:
Please add doc string for function which will explain about purpose, in args, return value etc